Make bootstrap doesn't exist

* carthage install method: [ ] .pkg, [ ] homebrew, [ X] source

Checked out master Carthage branch to see if the suggested fixes would work to get it building under Xcode 10.2 beta.

Looked at the contributing document to see how to start and executed `make bootstrap` which gave me this output:

```
Bit-Shifter:Carthage derekclarkson$ make bootstrap
make: *** No rule to make target `bootstrap'.  Stop.
```

Um. So now what? :-)

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -10,16 +10,13 @@ If you know exactly how to implement the feature being suggested or fix the bug 
 
 If you can’t make the change yourself, please open an issue after making sure that one isn’t already logged.
 
-## Target CarthageKit
-
-Unless you’re specifically improving something about the command-line experience of Carthage, please make code changes to [CarthageKit](README.md#carthagekit). This framework increases modularity, and allows other tools to integrate with Carthage more easily.
-
 ## Get started
 
 After checkout, you can run the following command from the cloned directory, and then open the workspace in Xcode:
 
 ```bash
-make bootstrap
+make xcodeproj && \
+  open Carthage.xcodeproj
 ```
 
 Then, to install your development copy of Carthage (and any local changes you've made) on your system, and test with your own repos:

--- a/Makefile
+++ b/Makefile
@@ -22,7 +22,7 @@ MKDIR=mkdir -p
 SUDO=sudo
 CP=cp
 
-.PHONY: all clean install package test uninstall
+.PHONY: all clean install package test uninstall xcconfig xcodeproj
 
 all: installables
 
@@ -66,3 +66,13 @@ install: installables
 
 uninstall:
 	$(RM) "$(BINARIES_FOLDER)/carthage"
+	
+.build/libSwiftPM.xcconfig:
+	mkdir -p .build
+	echo "OTHER_LDFLAGS = -lncurses -lsqlite3" > "$@"
+	echo "OTHER_CFLAGS = -DSWIFT_PACKAGE" >> "$@"
+
+xcconfig: .build/libSwiftPM.xcconfig
+
+xcodeproj: xcconfig
+	 swift package generate-xcodeproj --xcconfig-overrides .build/libSwiftPM.xcconfig

